app/View/Servers/edit.ctp

```diff
@@ -236,7 +240,8 @@ var formInfoValues = {
         'ServerPublishWithoutEmail' : '<?php echo __('Publish new event without email (working with Pull event).');?>',
         'ServerSubmittedCert' : "<?php echo __('You can also upload a certificate file if the instance you are trying to connect to has its own signing authority.');?>",
         'ServerSubmittedClientCert' : "<?php echo __('You can also upload a client certificate file if the instance you are trying to connect requires this.');?>",
-        'ServerSelfSigned' : "<?php echo __('Click this, if you would like to allow a connection despite the other instance using a self-signed certificate (not recommended).');?>"
+        'ServerSelfSigned' : "<?php echo __('Click this, if you would like to allow a connection despite the other instance using a self-signed certificate (not recommended).');?>",
+        'ServerRemoveMissingTags': "<?php echo __('Remove any global tags from events on local instance that are not present on an updated event being received from the server. This will remove any custom global tags you have set (working with Pull event).');?>"
```

Comment:
The description is great but it would be nice if it includes the fact that local tags are unaffected and that it only work for pulling and that pushing is not affected. (This is nitpicking but for a feature that only works in one way, I would prefer to be as transparent as possible)